### PR TITLE
Update BestOf.yml

### PR DESCRIPTION
**Pull request by aelfa**

### CODE REVIEW

**Comment by aelfa** [comment]
Syntax Fix!

File: Speedmod/Movies/BestOf.yml [hunk 1]
```diff
@@ -17,11 +17,6 @@ templates:
     visible_shared: true
 
 collections:
-
```
Comment (cpt-kuesel) [inline Speedmod/Movies/BestOf.yml:20]: Don‘t delete other peoples comments / structure

File: Speedmod/Movies/BestOf.yml [hunk 2]
```diff
@@ -249,18 +245,10 @@ collections:
...
     
-      Best Superhero Movies:
+  Best Superhero Movies:
         trakt_list: https://trakt.tv/users/hdlists/lists/best-superhero-movies
         sort_title: +++++++++++++++++++++_Superhero Movies
         sync_mode: sync
         smart_label: critic_rating.desc
         summary: Best Superhero Movies of All Time
         url_poster: https://i.imgur.com/lqXRQdd.png
-
```
Comment (cpt-kuesel) [inline Speedmod/Movies/BestOf.yml:259]: Don‘t delete other peoples collections

File: Speedmod/Movies/BestOf.yml [hunk 3]
```diff
@@ -55,7 +50,8 @@ collections:
     summary: Popular Movies across the internet
   Top Rated Global:
```
Comment (cpt-kuesel) [inline Speedmod/Movies/BestOf.yml:51]: This collection does not look fixed to me, review the indentation of the different builders again

**Review by cpt-kuesel — CHANGES_REQUESTED** [review]
If you submit fixes for other peoples files, please limit yourself to fixing and dont delete stuff out of the files.
I have left comments for you so please take a look and resubmit a fixes only pull request.

**Comment by aelfa** [comment]
> If you submit fixes for other peoples files, please limit yourself to fixing and dont delete stuff out of the files. I have left comments for you so please take a look and resubmit a fixes only pull request.

@cpt-kuesel I haven't removed a thing, The comments removed were a mistake I've rectified that in the new PR, Secondly, that last collection was referred two times in the same `yml` file. So one has to be removed and/or renamed.

**Comment by aelfa** [comment]
https://github.com/meisnate12/Plex-Meta-Manager-Configs/blob/7b8da07477f64f5036836690f15ad97539e6f20f/Speedmod/Movies/BestOf.yml#L158

**Comment by cpt-kuesel** [comment]
I see, I didnt look close enough there.
Thats okay then.
Did you still want to have this merged or did you close the pull request on purpose?

**Pull request closed without merging**